src/app/base/countLiquid.js

```diff
@@ -0,0 +1,24 @@
+const countLiquid = (container, setLitresData, setOptimalLitresData) => {
+    const liquidRate = container.querySelector('.js-liquidRate');
+    const liquidLowIcon = container.querySelector('.js-liquidLow');
+    const liquidNormalIcon = container.querySelector('.js-liquidNormal');
+    const optimalLitres = container.querySelector('.js-liquid-optimal');
+
+    optimalLitres.innerHTML = setOptimalLitresData;
+
+    if (setLitresData < setOptimalLitresData) {
+        liquidRate.innerHTML = 'insufficient';
+        if (liquidRate.classList.contains('range--hight')) liquidRate.classList.remove('range--hight');
```

Comment:
For `liquidRate.innerHTML = 'insufficient';` is faster to use `liquidRate.textContent = 'insufficient';` since you are replacing just `text` and not `HTML` tags